muspinsim_input/muspinsim_input.xml

```diff
@@ -102,69 +108,75 @@
                         </param>
                     </when>
                     <when value="dissipation">
-                        <param name="dis_index" type="integer" value="" label="Index of spin to be given dissipation term (index starts from 1)"/>
-                        <param name="dis_val" type="text" value="" label="Enter dissipation value (MHz)"
-                               help="Define dissipation term.
+                        <param name="dis_index" type="integer" value="" label="Index of spin with dissipation"
+                               help="Index refers to the position of the spin as selected in 'spins' section"/>
+                        <param name="dis_val" type="text" value="" label="Dissipation"
+                               help="Define dissipation term (MHz).
                                Allows default expressions, constants and functions (see help)"/>
                     </when>
                 </conditional>
             </repeat>
         </section>
         <section name="experiment_params" expanded="true" title="Experiment Parameters">
             <conditional name="experiment">
-                <param name="experiment_preset" type="select" value='custom' display="radio" label="Choose an experiment type, or custom to define own params"
-                       help="`alc` - Avoided Level Crossing,
-                             `zero field` - No external,
-                             `custom` - No preset values fields,
-                             (See muspinsim docs 'experiment' keyword for more info)
-                            ">
+                <param name="experiment_preset" type="select" value='custom' display="radio" label="Experiment type"
+                       help="Avoided Level Crossing (ALC) - polarization: longitudinal, x-axis: field, y-axis: integral
+                             Zero field - polarization: transverse, field: 0
+                             Custom - No preset">
```

Comment:
Remember newlines in the hint text don't get rendered in the UI, so extra punctuation is needed. I find the dictionary style structure here quite difficult to parse and understand here as well.